openmetadata-ui/src/main/resources/ui/src/components/Settings/Services/AddService/AddService.component.tsx

```diff
@@ -0,0 +1,326 @@
+/*
+ *  Copyright 2022 Collate.
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import { Space, Typography } from 'antd';
+import { AxiosError } from 'axios';
+import { t } from 'i18next';
+import { capitalize, isEmpty, isUndefined } from 'lodash';
+import { LoadingState } from 'Models';
+import React, { useEffect, useState } from 'react';
+import { useHistory } from 'react-router-dom';
```

Comment:
We have removed this component. Let's be aware of the changes we make while resolving the conflicts